src/traces/sankey/plot.js

```diff
@@ -124,7 +124,8 @@ module.exports = function plot(gd, calcData) {
     };
 
     var linkHoverFollow = function(element, d) {
-
+        var trace = gd._fullData[d.traceId];
+        var ptNumber = d.originalIndex;
```

Comment:
We can leave it off for now to get 1.27 out, but we should definitely include custom hovertext (and at that point we might as well include custom styling) for both nodes and links in the near future - I can definitely see people wanting to explain what specifically is included in each of these elements in more detail than is displayed permanently on the diagram. Chatting with @monfera seems like `link.label` fills this role for links but there's no analog for nodes. I'll make an issue for this so we don't forget about it.